lib/reportDisables.js

```diff
@@ -25,19 +25,21 @@ module.exports = function (results) {
 
 		const config = result._postcssResult.stylelint.config;
 
+		if (!config || !config.rules) return;
+
 		// If no rules actually disallow disables, don't bother looking for ranges
 		// that correspond to disabled rules.
-		if (!Object.values(_.get(config, 'rules', {})).some(reportDisablesForRule)) {
-			return [];
+		if (!Object.values(config.rules).some(reportDisablesForRule)) {
+			return;
 		}
 
-		Object.keys(rangeData).forEach((rule) => {
-			rangeData[rule].forEach((range) => {
-				if (!reportDisablesForRule(_.get(config, ['rules', rule], []))) return;
+		for (const rule of Object.keys(rangeData)) {
+			for (const range of rangeData[rule]) {
```

Comment:
[suggestion] Same here:

```suggestion
		for (const [rule, ranges] of Object.entries(rangeData)) {
			for (const range of ranges) {
```